File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using DevExpress.XtraEditors;
using DevExpress.Skins;
using WindowsApplication1.Properties;
using DevExpress.LookAndFeel;

namespace WindowsApplication1 {
    public partial class Form1 : XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void ChangeSkinElement() {
            SetElementImage(GetCurrentSkin(), Resources.formbuttonhelp);
            LookAndFeelHelper.ForceDefaultLookAndFeelChanged();
        }

        private Skin GetCurrentSkin() {
            return FormSkins.GetSkin(defaultLookAndFeel1.LookAndFeel.ActiveLookAndFeel);
        }
        public void SetElementImage(Skin currentSkin, Image image) {
            var element = currentSkin[FormSkins.SkinFormButtonClose];
            element.SetActualImage(image, true);
        }
        private int i;
        private void simpleButton1_Click(object sender, EventArgs e) {
            i++;
            var index = i % SkinManager.Default.Skins.Count;
            defaultLookAndFeel1.LookAndFeel.SkinName = SkinManager.Default.Skins[index].SkinName;
            if (checkEdit1.Checked) {
                ChangeSkinElement();
            }
            Text = defaultLookAndFeel1.LookAndFeel.SkinName;
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.Skins
Imports DevExpress.LookAndFeel

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub ChangeSkinElement()
            SetElementImage(GetCurrentSkin(), Global.My.Resources.formbuttonhelp)
			LookAndFeelHelper.ForceDefaultLookAndFeelChanged()
		End Sub

		Private Function GetCurrentSkin() As Skin
			Return FormSkins.GetSkin(defaultLookAndFeel1.LookAndFeel.ActiveLookAndFeel)
		End Function

		Public Sub SetElementImage(ByVal currentSkin As Skin, ByVal image As Image)
			Dim element As SkinElement = currentSkin(FormSkins.SkinFormButtonClose)
			element.Image.Image = image
		End Sub

		Private i As Integer

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			i += 1
			Dim index As Integer = i Mod SkinManager.Default.Skins.Count
			defaultLookAndFeel1.LookAndFeel.SkinName = SkinManager.Default.Skins(index).SkinName
			If checkEdit1.Checked Then
				ChangeSkinElement()
			End If
			Text = defaultLookAndFeel1.LookAndFeel.SkinName
		End Sub
	End Class
End Namespace